taxi/forms.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class DriverCreateForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "username",
```

Comment:
The `username` field is already included in `UserCreationForm.Meta.fields`. Including it again here is redundant and should be removed to avoid duplication.